src/ui/SubtitleEdit.csproj

```diff
@@ -284,18 +284,4 @@
     <Compile Remove="Logic\VideoPlayers\DirectShowLib2005.cs" />
     <EmbeddedResource Remove="Forms\GetDictionaries.resx" />
   </ItemGroup>
-  <Target Name="BeforeCompile">
```

Comment:
Plus this target is not valid anymore

https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-targets?view=vs-2022#sdk-style-projects

> In MSBuild, custom targets need to be explicitly invoked or included in the build process.